code/game/objects/effects/decals/Cleanable/tar.dm

```diff
@@ -0,0 +1,47 @@
+/obj/effect/decal/cleanable/tar
+	gender = PLURAL
+	name = "tar"
+	desc = "A sticky substance."
+	icon_state = "tar2"
+	density = FALSE
+	opacity = FALSE
+	var/turf/simulated/target
+	var/old_slowdown
+
+/obj/effect/decal/cleanable/tar/New(turf/simulated/loc)
+	if(!istype(loc, /turf/simulated) || loc.has_tar) // Check if the turf already has tar
+		qdel(src)
+		return
```

Comment:
do you need to perform this check? It seems like this is all handled in `reaction_turf` already